src/hermes.jsx

```diff
@@ -0,0 +1,338 @@
+'use strict';
+import React, {PropTypes} from 'react';
+import debounce from 'debounce';
+import Flyout from '@streammedev/flyout';
+import {HermesWrapper} from './wrapper';
+import defaultFormatValue from './default-format-value';
+import ifPropCall from './if-prop-is-func-call';
+const selection = typeof window !== 'undefined' ? require('selection-range') : () => {};
+const nbsp = String.fromCharCode(160);
+const zws = String.fromCharCode(8203);
+// Track which placeholder css rules have been added for the hermes instances, see below
+const cssRules = {};
+
+export const Hermes = React.createClass({
```

Comment:
Would prefer `export default class Hermes extends Component({` since we're rolling so deep in ES6.